templates/next/app/page.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  FrameButton,
+  FrameContainer,
+  FrameImage,
+  FrameReducer,
+  NextServerPageProps,
+  getFrameMessage,
+  getPreviousFrame,
+  useFramesReducer,
+} from "frames.js/next/server";
+
+type State = {
+  saidGm: boolean;
+};
+
+const initialState: State = { saidGm: false };
+
+const reducer: FrameReducer<State> = (state, action) => {
+  return {
+    saidGm: true,
+  };
+};
+
+// This is a react server component only
+export default async function Home({
+  params,
+  searchParams,
+}: NextServerPageProps) {
+  const previousFrame = getPreviousFrame<State>(searchParams);
+
+  const frameMessage = await getFrameMessage(previousFrame.postBody, {
+    // remove if you aren't using @frames.js/debugger or you just don't want to use the debugger hub
```

Comment:
how does this work with deploying to production?

Reply:
Hm that would need a check if you are running on `NODE_ENV=production` and remove it in that case, if that's what is on your mind.

Reply:
Added the condition

Reply:
@davidfurlong is this ok with you?